### ISSUE TITLE
I can get automaticaly to the first page, i always get the callback url

### ISSUE THREAD

**Issue by Frenchy75** [microsoft/dicom-ohif#7]
When I browse the storage web endpoint, my browser show the call back with the token in the URL, I then have to change the url to the endpoint so I can access the viewer.

**Comment by StevenBorg** [comment]
@Frenchy75, I'm seeing that, too. Looks like there's a repeatable bug in the way OHIF handles redirection... we think.  Or it could be in our configuration somewhere. We're looking into it, because it's a pain and doesn't lend much confidence to the solution. :-)

**Comment by smithago** [comment]
I have created a bug for ohif https://github.com/OHIF/Viewers/issues/2850

**Comment by smithago** [comment]
Worksaround is to remove the callbackurl and retry again.

**Comment by abymmathew-mli** [comment]
For me, it was stemming from the fact that it's (OIDC client) using the token_id to extract the bearer token. Hence in the configuration, we need to add 

response_type: 'token id_token', // "implicit"
loadUserInfo: false

**Comment by alphacentauridigital** [comment]
Hello,

We have integrated the OHIF viewer with the angular app.  When we try to open the viewer using the link to open in a new tab. Even though the user is logged in to OHIF viewer once. It always redirects to the callback URL with a token as a query parameter. After inspecting cookies, even though OHIF viewer is preserving redirect URLs in cookies. It is not being used.

Tried abymmathew's solution but didn't work.

Has someone resolved this issue?

Regards
